### ISSUE TITLE
How to use loadedFrom and resolve?

### ISSUE THREAD

**Issue by timothymcmackin** [Mermade/widdershins#176]
I'm trying to copy my schemas into a separate file and refer to them with `"$ref": "mySchemas.json#/components/schemas/mySchema"`. I've set the Widdershins `resolve` parameter to true and I've tried various values for `loadedFrom`, but Widdershins can't seem to load the schemas from the file. The body parameter is empty in the MD output. I'm doing the conversion locally (the files are not on a server); could that be the problem, or am I doing something else wrong?

I've tried this by editing the petstore example:
https://gist.github.com/timothymcmackin/29776d56a654e0a8cccda18a3213d774

Here's how I've linked to the schema file:

```
"paths": {
    "/pet": {
        "post": {
            "summary": "Add a new pet to the store",
            "description": "",
            "operationId": "addPet",
            "requestBody": {
                "$ref": "petstore3_schemas.json#/components/requestBodies/Pet"
            }
        },
        ...
```

I get this output:
```
`POST /pet`

*Add a new pet to the store*

> Body parameter

```json
null

...
```

**Comment by timothymcmackin** [comment]
I've tried hosting my files locally and setting the swagger file site param and the widdershins loadedFrom param to `http://localhost:9080`, but the output is the same.

**Comment by MikeRalphson** [comment]
I've made a small change to the handling of `options.verbose` to make debugging this easier, and added a clarification to the `README`. These changes are in `v3.6.4`

`options.loadedFrom` has no bearing on `options.resolve`. `loadedFrom` is only required when the source document doesn't declare any `servers` (or `schemes`, `host`, `basePath`) and you want a specific API endpoint constructed in the output markdown.

The option you want is `source`. When `options.resolve` is `true`, `options.source` must be set to the **full** path or URL of the document you want to be resolved. External `$ref`s will then be resolved relative to that `source`.

That should help, but please chase up this issue if not. Thanks for pointing it out.

**Comment by timothymcmackin** [comment]
@MikeRalphson That makes sense, thanks. I suggested a clarification in https://github.com/Mermade/widdershins/pull/177 . 

I've got the resolution of $refs working in my project now, thanks. One thing that changes when I use external refs is that I can no longer get the name of the schema or a link to it. `{{=p.safeType}}` returns `object` instead of `#mySchemaId`. I don't suppose you know of any way to get the ID of the schema or a link to it? I've tried variations of `expandBody` and `omitBody` but the result doesn't change.

I ask because I'm using Widdershins to convert several Swagger files to MD and then concatenating them into a single file before I send it to Shins. My plan was to consolidate my schemas into a single file and append that to the end, so I'm not showing a duplicated schema section after each chunk of Swagger content. So I'd like to link to the schemas at the end of the document.
